storage_v2/factoryadapter/reader_test.go

```diff
@@ -62,9 +62,11 @@ func TestTraceReader_GetTraceDelegatesSuccessResponse(t *testing.T) {
 	traceReader := &TraceReader{
 		spanReader: sr,
 	}
-	traces, err := iter.CollectWithErrors(traceReader.GetTrace(
```

Comment:
is `CollectWithErrors` still being used?

Reply:
don't think so

Reply:
maybe not

Reply:
can we remove it then?

Reply:
It's not part of this PR, so I'd leave it. That whole package will go away in a couple of months.